src/test/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSparkUnitTest.java

```diff
@@ -71,13 +71,8 @@ public void getQNamesTest() {
 
     @Test(groups = "spark")
     public void getKmerIntervalsTest() {
-        final HopscotchSet<SVKmer> killSet = new HopscotchSet<>(2);
-        killSet.add(SVKmerizer.toKmer("AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"));
-        killSet.add(SVKmerizer.toKmer("CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"));
         final List<SVKmer> highCountKmers = FindBreakpointEvidenceSpark.getHighCountKmers(params, reads, locations, null);
-        Assert.assertEquals(highCountKmers.size(), 2);
-        killSet.addAll(highCountKmers);
-        Assert.assertEquals(killSet.size(), 2);
+        Assert.assertEquals(highCountKmers.size(), 0);
```

Comment:
Are you still testing that `getHighCountKmers` can return some high count kmers somewhere?